page_objects/license_details_page.py

```diff
@@ -0,0 +1,49 @@
+from utilities.ui_utilities.base_page import BasePage
+from selenium.webdriver.common.by import By
+
+
+class LicenseDetailsPage(BasePage):
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    __license_header = (By.XPATH, "//h1[contains(text(), 'License Details')]")
+    __delete_button = (By.XPATH, "//*[@type='danger-alt']")
+    __cancel_delete_button = (By.XPATH, "//*[contains(text(),'Cancel')]")
+    __remove_license_key_button = (By.XPATH, "//*[contains(text(),'Remove license key')]")
+    __basic_mode_label = (By.XPATH, "//*[contains(text(),'basic mode')]")
+    __type_value = (By.XPATH, "//label[contains(text(),'Type')]//following-sibling::*/div/label")
+    __buy_license_button = (By.XPATH, "//div[contains(text(), 'BUY LICENSE')]")
+    __go_to_directory_button = (By.XPATH, "//div[contains(text(), 'GO TO')]")
+    __subscribe_dialog_header = (By.XPATH, "//h2[contains(text(),'Subscribe')]")
+
+    def is_license_header_shown(self):
+        return self.is_displayed(self.__license_header)
+
+    def invoke_delete_license(self):
+        self.click(self.__delete_button)
+        self.is_displayed(self.__remove_license_key_button)
```

Comment:
what purpose of this line?

Reply:
initially to make sure pop up is shown. but yeah, waiter is handled by click. - fixed